### DISCUSSION TITLE
Mesh problem

### DISCUSSION THREAD

**Discussion by torodecamargue** [nextfoam/baram#77] [General]
Hello,

During snap operation I have some deviation of the mesh at interface. Do you know how to imlprove this king of issue?

![image](https://github.com/nextfoam/baram/assets/75060318/227145b4-58a2-4bd1-9919-5a6ebecb115a)

I suspect that it gives me the following error message during first step of computation

![image](https://github.com/nextfoam/baram/assets/75060318/42ba1ed3-9f35-4af8-b552-f528f66d2569)

Best regards

Julien

**Comment by torodecamargue** [comment]
I try 10 or 12 different meshes and always I finished with this message 

![image](https://github.com/nextfoam/baram/assets/75060318/05ca6958-ee65-4a8f-96f8-567dcc16dda5)

I don't find where I can increase the toelance with 'matchTolerance' command. Maybe with a better snap during the mesh operation it would be even better.

Best regards

**Comment by kdg5424** [comment]
Hi. This is Donggyu.

We usually recommend **"Feature Snap Type : implicit**
You could change at the snap operation.
If you have the problem again, please let us know.

Thanks.

**Comment by torodecamargue** [comment]
Hello Donggyu,

I have already try this settings, I think that the interface aspect is improved visually but the error message still happens.

Best regards
